website/docs/r/gcp_source.html.markdown

```diff
@@ -0,0 +1,50 @@
+---
+layout: "sumologic"
+page_title: "SumoLogic: sumologic_gcp_source"
+description: |-
+  Provides a Sumologic GCP Source.
+---
+
+# sumologic_gcp_source
+Provides a [Sumologic GCP Source][2].
```

Comment:
Can we use the full name (not abbreviation) here? (should at least use the full name once on this page)

Reply:
Also, `Sumo Logic` not `Sumologic` probably other pages have this wrong as well